contracts/wildcredit/AnteWildCreditOracleTest.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+
+// ┏━━━┓━━━━━┏┓━━━━━━━━━┏━━━┓━━━━━━━━━━━━━━━━━━━━━━━
+// ┃┏━┓┃━━━━┏┛┗┓━━━━━━━━┃┏━━┛━━━━━━━━━━━━━━━━━━━━━━━
+// ┃┗━┛┃┏━┓━┗┓┏┛┏━━┓━━━━┃┗━━┓┏┓┏━┓━┏━━┓━┏━┓━┏━━┓┏━━┓
+// ┃┏━┓┃┃┏┓┓━┃┃━┃┏┓┃━━━━┃┏━━┛┣┫┃┏┓┓┗━┓┃━┃┏┓┓┃┏━┛┃┏┓┃
+// ┃┃ ┃┃┃┃┃┃━┃┗┓┃┃━┫━┏┓━┃┃━━━┃┃┃┃┃┃┃┗┛┗┓┃┃┃┃┃┗━┓┃┃━┫
+// ┗┛ ┗┛┗┛┗┛━┗━┛┗━━┛━┗┛━┗┛━━━┗┛┗┛┗┛┗━━━┛┗┛┗┛┗━━┛┗━━┛
+// ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+// ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+pragma solidity >=0.8.0;
+
+import "../AnteTest.sol";
+
+interface IPriceOracle {
+    function tokenPrice(address _token) external view returns (uint256);
+}
+
+// Ante Test to check if the Wild Credit $WILD token price is always within 1% of the UniswapV3Price oracle
```

Comment:
Is the oracle price of $WILD used anywhere in the underlying protocol? Seems like the tokens being lent and borrowed are more critical to test that the oracle prices remain accurate, but I may not understand the $WILD tokenomics well enough

Reply:
TokenPrice is used for the PairFactory contract, you are right that I should add additional tests for the available pools atm.
https://etherscan.io/address/0x23b74796b72f995e14a5e3ff2156dad9653256cf#code

```
  function _convertTokenValues(
    address _fromToken,
    address _toToken,
    uint    _inputAmount
  ) internal view returns(uint) {

    uint priceFrom = controller.tokenPrice(_fromToken) * 1e18 / 10 ** IERC20(_fromToken).decimals();
    uint priceTo   = controller.tokenPrice(_toToken)   * 1e18 / 10 ** IERC20(_toToken).decimals();

    return _inputAmount * priceFrom / priceTo;
  }
 ```
 
This is where token prices are used. There are actually 55 lending pools. I think because they are single sided, the number is half, maybe 1 pool is for wild token. I need to check.

Reply:
$WILD is a token within the oracle contract so I used it initially. I'll narrow it down to a few key pools that are used in the PairToken contract in my next commit. Can't loop through 55 pools. There are overlaps too.

Reply:
I have found all the key pools in the reward distribution contract, it does seem that $WILD isn't used, but it may be used in future. The $WILD token is added to the Oracle contract so I'll also test it just in case.

Tokens used, this has been added in the latest commit
```
    // these are the tokens used in wild credit rewardDistribution
    address[] tokenList = [
        0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2, // WETH
        0x6B175474E89094C44Da98b954EedeAC495271d0F, // DAI
        0xBB0E17EF65F82Ab018d8EDd776e8DD940327B28b, // AXS
        0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0, // MATIC
        0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9, // AAVE
        0xc00e94Cb662C3520282E6f5717214004A7f26888, // COMP
        0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F, // SNX
        0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2, // MKR
        0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599, // WBTC
        0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984, // UNI
        0x514910771AF9Ca656af840dff83E8264EcF986CA, // LINK
        0xD533a949740bb3306d119CC777fa900bA034cd52, // CRV
        0xbC396689893D065F41bc2C6EcbeE5e0085233447, // PERP
        0x111111111117dC0aa78b770fA6A738034120C302, // 1INCH
        0x0391D2021f89DC339F60Fff84546EA23E337750f, // BOND
        0x08A75dbC7167714CeaC1a8e43a8d643A4EDd625a // WILD (this is not used in reward distribution and lending pairs but is in the oracle contract, we add it here just in case)
    ];
```

Reply:
Looks great, thanks! Nice job tracking down all the tokens